pages/maker/index.js

```diff
@@ -2,22 +2,28 @@ import React, { useEffect } from 'react'
 import { Flex, Box, Type } from 'blockstack-ui'
 import { connect } from 'react-redux'
 
-import { selectMaker } from '@stores/maker/reducer'
-import { fetchApps } from '@stores/maker/actions'
+import { selectMaker, selectAppList, selectCurrentApp } from '@stores/maker/reducer'
```

Comment:
Let's try and follow what we've been doing previously, and have all selectors inside `@stores/maker/selectors`

Reply:
I'll change this, and address the other comments tomorrow.